src/Components/SliderButton/SliderButton.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import "./SliderButton.css";
+
+export const SliderButton = ({ className }) => (
+  <div className={className}></div>
+);
```

Comment:
Do you need this component with reduced data?

Reply:
Added swiper for displaying all the cards  from the array.